### PR TITLE
conic solver, unknown is approximate optimal

### PR DESCRIPTION
**Pull request by Mojusko**
## Description
I added the UNKNOWN into approximate solution.

### CODE REVIEW

File: cvxpy/reductions/solvers/solver.py [hunk 1]
```diff
@@ -68,14 +68,14 @@ def is_installed(self) -> bool:
...
 
     @abc.abstractmethod
-    def solve_via_data(self, data, warm_start: bool, verbose: bool, solver_opts, solver_cache=None):
+    def solve_via_data(self, data, warm_start: bool, verbose: bool,certified_soln: bool, solver_opts, solver_cache=None):
         """Solve a problem represented by data returned from apply.
         """
         raise NotImplementedError()
 
-    def solve(self, problem, warm_start: bool, verbose: bool, solver_opts):
+    def solve(self, problem, warm_start: bool, verbose: bool, certified_soln: bool, solver_opts):
```
Comment (SteveDiamond) [inline cvxpy/reductions/solvers/solver.py:76]: Same as above, this should be in solver_opts not a new arg.

File: cvxpy/settings.py [hunk 2]
```diff
@@ -48,6 +48,7 @@
 # Solver Constants
 OPTIMAL = "optimal"
 OPTIMAL_INACCURATE = "optimal_inaccurate"
+NOTCERTIFIED = 'unknown'
```
Comment (SteveDiamond) [inline cvxpy/settings.py:51]: I don't think there should be a new status. Map unknown to optimal_inaccurate instead.

**Comment by Mojusko** [comment]
Will do indeed, cheers for feedback.

**Comment by the-moliver** [comment]
@Mojusko Curious if you plan to finish this up at it would be most useful to me :-)

**Comment by rileyjmurray** [comment]
This PR has been subsumed by #2117, which should be merged soon.

**Pull request closed without merging**